packages/react/package.json

```diff
@@ -22,13 +22,16 @@
     "jwt"
   ],
   "type": "module",
-  "types": "dist/main.d.ts",
```

Comment:
Given that two projects are now being used in a single build process, `tsc` outputs two folders with type declarations in `dist`.